CHANGELOG.md

```diff
@@ -1,4 +1,18 @@
-## v1.1.0 - 2024-04-30
+# Changelog
```

Comment:
I think these updates to the change log are no longer consistent with the `bin/extract-release-notes.sh` script.

The format of Keep A Change Log is cool, but the script to extract the release notes would have to change to accommodate these changes. (One bit of it would be something like `VERSION_PATTERN="^## \[v"` - I think there are a few other bits, too).

Reply:
That is probably true. Dang.  Let me check that.

Reply:
[6285d4c](https://github.com/nasa/harmony-browse-image-generator/pull/14/commits/6285d4cb2dfbd589591436d5ed6fc27e5b0c5029)

Reply:
It would break if there was only one entry in the file. Or at least it would include the links. Is that what you were thinking?

Reply:
I ran the updated script locally, and I think this looks good. We can probably cope with the assumption that this file already has prior release notes in it to make life easier.

Reply:
I could probably figure this out. But happy to leave it as just working.

Reply:
Ok, I can make it work generally. [1f858ef](https://github.com/nasa/harmony-browse-image-generator/pull/14/commits/1f858ef0a11ebcad1f5688053db1b2ec6dfa3b82)